packages/office-ui-fabric-react/src/components/Button/CommandBarButton/CommandBarButton.styles.ts

```diff
@@ -29,34 +29,22 @@ export const getStyles = memoizeFunction((
 
     rootHovered: {
       backgroundColor: theme.palette.neutralLight,
-      color: theme.palette.neutralDark,
-      icon: {
```

Comment:
@micahgodbolt Here. Having "icon" here doesn't make any sense and violates the typings. In glamor I believe it would consider this a selector (as in "icon" html element type.)